playbooks/update-dat.yml

```
@@ -28,14 +28,16 @@
         msg: |
           "release_url: https://github.com/techprober/v2ray-rules-dat/releases/tag/{{ latest_release.stdout }}"
 
-    - name: Download dat files with the latest release
+    - name: Download rule files with the latest release
       ansible.builtin.get_url:
-        url: "{{ item }}"
+        url: "https://github.com/techprober/v2ray-rules-dat/releases/download/{{ latest_release.stdout }}/{{ item }}"
         dest: "{{ dest_path }}"
       loop:
-        -  "https://github.com/techprober/v2ray-rules-dat/releases/download/{{ latest_release.stdout }}/geosite.dat"
-        -  "https://github.com/techprober/v2ray-rules-dat/releases/download/{{ latest_release.stdout }}/geoip.dat"
-    
+        - "geosite.dat"
+        - "geoip.dat"
+        - "reject-list.txt"
+        - "direct-list.txt"
```

Based on the upstream - https://github.com/techprober/v2ray-rules-dat/releases/, `reject-list.txt` is not available in release. Basically, `direct-list.txt` and `reject-list.txt` are custom lists that are managed by the end-user directly. Hence, we should get rid of them here.

```suggestion
      loop:
        - "geosite.dat"
        - "geoip.dat"
```